load-test/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: load-test
+description: A Helm chart for Kubernetes
```

Comment:
could you add a short description that explains the objective of simulate targets

Reply:
I'll add as well a short readme